Lesson7.ipynb

```diff
@@ -29,6 +33,551 @@
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>Total Steps</th>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Team Name</th>\n",
+       "      <th></th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>TBD</th>\n",
+       "      <td>1820997.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Stop when you drop!!</th>\n",
+       "      <td>1721534.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>The Slimsons</th>\n",
+       "      <td>1484755.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Scrambled Legs</th>\n",
+       "      <td>1480327.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Tears for Beers</th>\n",
+       "      <td>1468397.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>11,111 to No. 1</th>\n",
+       "      <td>1395549.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Cereal Killers</th>\n",
+       "      <td>1333977.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>FuFu</th>\n",
+       "      <td>1198137.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Will Walk for Wine</th>\n",
+       "      <td>1158732.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Just Step It</th>\n",
+       "      <td>1128860.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Here come the hotsteppers</th>\n",
+       "      <td>1107342.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Smarty Pints</th>\n",
+       "      <td>1055281.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Sole Striders</th>\n",
+       "      <td>954382.0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                           Total Steps\n",
+       "Team Name                             \n",
+       "TBD                          1820997.0\n",
+       "Stop when you drop!!         1721534.0\n",
+       "The Slimsons                 1484755.0\n",
+       "Scrambled Legs               1480327.0\n",
+       "Tears for Beers              1468397.0\n",
+       "11,111 to No. 1              1395549.0\n",
+       "Cereal Killers               1333977.0\n",
+       "FuFu                         1198137.0\n",
+       "Will Walk for Wine           1158732.0\n",
+       "Just Step It                 1128860.0\n",
+       "Here come the hotsteppers    1107342.0\n",
+       "Smarty Pints                 1055281.0\n",
+       "Sole Striders                 954382.0"
+      ]
+     },
+     "execution_count": 202,
+     "metadata": {},
+     "output_type": "execute_result"
```

Comment:
The output table doesn't look like what it should do.
No "Date" column and "Team Name" and "Total Steps" are not aligned.